keras_cv/training/contrastive/contrastive_trainer.py

```diff
@@ -44,10 +44,12 @@ class labels at train-time using the encoder output as input.
 
     Usage:
     ```python
-    encoder = keras_cv.models.DenseNet121(
-        include_rescaling=True,
-        include_top=False,
-        pooling="avg")
+    encoder = keras.Sequential(
+                  [
```

Comment:
This indenting to too deep. Every extra indent should be four spaces only.

Reply:
Fixed